akka-docs/src/test/scala/docs/stream/StreamTestKitDocSpec.scala

```diff
@@ -29,9 +29,6 @@ class StreamTestKitDocSpec extends AkkaSpec {
 
   "grouped part of infinite stream" in {
     //#grouped-infinite
-    import system.dispatcher
```

Comment:
Were not used by the code below? Interesting. Since it was an example for our docs.

Reply:
Yeah, maybe someone thought it was needed, not sure